examples/tensorflow/common/object_detection/ops/spatial_transform_ops.py

```diff
@@ -11,6 +11,7 @@
  limitations under the License.
 """
 
+from tensorflow.keras import layers
```

Comment:
I'm not sure it is really necessary Why we can not use `tf.keras.layers`?

Reply:
Sure, updated!